packages/at_chat_flutter/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ version: 3.0.8
 homepage: https://docs.atsign.com/
 repository: https://github.com/atsign-foundation/at_widgets/tree/trunk/packages/at_chat_flutter
 issue_tracker: https://github.com/atsign-foundation/at_widgets/issues
-documentation: https://atsign.dev/docs/functional_architecture/libraries/#at_chat_flutter
+documentation: https://atsign.gitbook.io/docs/~
```

Comment:
Let's not link this documentation... this is a staging site.
I think we can remove the documentation link without losing any points, since we have homepage and repository links.

Reply:
got it will do.